src/libraries/System.Reflection.Emit/src/Resources/Strings.resx

```diff
@@ -288,4 +288,16 @@
   <data name="InvalidOperation_TokenNotPopulated" xml:space="preserve">
     <value>MetadataToken for the member is not generated until the assembly saved.</value>
   </data>
+  <data name="Argument_BadSigFormat" xml:space="preserve">
```

Comment:
`Argument_BadSigFormat` doesn't appear to be used.

Reply:
This is referenced in `SymbolType`, the message exist in CoreLib resources but not in ref emit